### PR TITLE
Add properties order in transform info.

### PR DESCRIPTION
**Pull request by khamer1410**
Add info about properties order in transform.

#### Summary

Add info about the importance of the order when using Z translation

#### Motivation

I found that my code is not working and debugg it by reading an external article. I believe this should be in the MDN doxs

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/css/transform-function/translatez/index.md [hunk 1]
```diff
@@ -126,6 +126,8 @@ Note if the `perspective()` value is less than the `translateZ()` value, such as
 further than the user's viewport. The smaller the difference between the perspective and translateZ values, the closer
 the user is to the element and the larger the translated element will seem.
 
+Note that `perspecitve` must be placed before `translateZ` - other way around it won't work.
```
Comment (teoli2003) [inline files/en-us/web/css/transform-function/translatez/index.md:129]: I think this is better:
```suggestion
> **Note:** As the composition of transforms isn't commutative, the order you write the different functions is significant. In particular, in general, you want `perspective()` to be placed before `translateZ()`.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by khamer1410** [comment]
Sorry, I missed the notification. Committed suggestion, feel free to edit and merge as You want.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**